en/06_people/05_journey_of_your_career.md

```diff
@@ -0,0 +1,68 @@
+
+I like lists of three's, short enough to remember and long enough to get some good direction.
+
+#### Short Version
+
+1. Define yourself
+2. Pick an audience
+3. Communicate regularly
+
+#### Define Yourself
+
+This is probably the hardest step, for a simple reason, everyone hates tradeoffs. Very important point, all strategy requires explicit (very clear, no ambiguity) or implicit (not defined, but understood by all) tradeoffs. If there are no tradeoffs, there is a very high likelihood of failing at everything. No one can be good at, or do, everything!
+
+Example: "I work in the public cloud." VS "I am a Google Cloud Platform expert, specializing in high level project architecture and staying informed of the ever changing offerings Google creates, discards, or re-brands."
+
+Try creating a few of these statements, feel them out, see how they fit you.
+
+Some things to consider when making these:
+
+* Ensure you enjoy working in your target area, you may be spending a lot of time there
+* Check to see if there is work where you are aiming!
+    * Wanting to be the best ALGOL developer may not make sense
+* Make sure it aligns with your career ladder somehow
+* Your organization Leveling Guide
+* Look at the market, know what it values in your space and keep it in mind
+    * Expertise in Excel may be good, but expertise in VBLite for Excel may be better
+* After deciding your brand, and who you are, look for proof. If you are claiming something, how can you prove it? Consider something like a Brag Doc, items from your quarterly reviews, things you have received Props Points for, actions people gave you kudos for, etc.
+
+Now you should have a good idea of who you are and what you want to share. This takes time and energy! Don't worry too much about getting it exact, everyone re-brands all of the time. If you turn out not like it or it does not work out, try again with something else!
+
+#### Pick an Audience
+
+With a strategy and evidence, you are ready to share! How do you get traction and figure out who to share what with?
+
+Start with your Pod Lead or Direct Manager, they have the largest ability to affect your work and the most interest in ensuring everyone knows how good you are. They can be your partner is refining your brand and getting the word out. Hint, this is all great content for your 1on1's!
+
+Engage and communicate with the Tech Lead on your project. They should know and be aware of the great things you are doing! They may or may not be in your organization or leadership, but it is still important they know you! Make sure they understand where you are knowledgeable.
+
+The Client Engagement person on your project should know you are someone they can call on for help and to answer questions in the area you are an expert in.
+
+Create influence everywhere you can!
```

Comment:
The purpose of this section is to pick an audience, and this bit of advice seems to contradict that.